DZAI/compile/fn_selectRandom.sqf

```diff
@@ -18,5 +18,5 @@ if(count _this > 0) then
     _ret = count _this;           //number of elements in the array
     _ret = floor (random _ret);   //floor it first 
     _ret = _this select _ret;     //get the element, return it
-};
+} else {_ret = _this;};
```

Comment:
If you're getting the _ret = undefined, it's because you're supplying fn_selectRandom with no paramaters
i.e:
`[] call BIS_fnc_selectRandom;` would return:
`Error in expression <ret = _this select _ret;           
};

_ret;>
  Error position: <_ret;>
  Error Undefined variable in expression: _ret
File z\addons\dayz_code\compile\BIS_fnc\fn_selectRandom.sqf, line 24`